prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -269,6 +290,19 @@ presubmits: # runs on PRs
     extra_refs:
       - <<: *test_infra_ref
         base_ref: release-0.9
+        
+  - name: pre-master-kyma-xip-integration
+    branches:
+       - master
+    <<: *gke_xip_job_template
+    labels:
+      preset-build-pr: "true"
+      <<: *gke_job_labels_template
+    extra_refs:
+      - <<: *test_infra_ref
+        base_ref: master
+    trigger: "/test pre-master-kyma-xip-integration"
```

Comment:
The `trigger` and `rerun_command` is not needed, because it is the same as a default one.